ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -241,34 +239,11 @@ public CompletableFuture<PutResult> put(NamedBlobRecord record) {
             } else {
               statement.setTimestamp(5, null);
             }
+            statement.setLong(6, buildVersion());
+            statement.setString(7, NamedBlobState.READY.name());
```

Comment:
Are there still plans to support a Put without overwrite option? If so, what would the query for that look like given timestamp based version ids?

Reply:
Good point, think it's doable, I will think about and add that.